data_processing/cleaning_demographic_data.Rmd

```diff
@@ -52,11 +50,412 @@ demo_dat %>%
+                                                  is.na(discharge_from_institution_in_the_last_six_months) ~ "unknown",
+                                                  TRUE ~ discharge_from_institution_in_the_last_six_months))
+```
+
+Check for discharge from institution
+```{r}
+demo_dat %>% 
+  count(discharge_from_institution_in_the_last_six_months, discharge_institution_recode)
+```
+
+APS report date - 
+
+### Question (for Sara): How do I code that we will want to check that the date is something between start of the program and the present? 
+
+Reporting Source - comining unknown responses into a "unknown" (including not applicable). Correcting spelling of "self", changing "client" to "self". Recoding "relative" and "family" as "family member".Changing "community professional" to "other community professional" to match their drop down. Changing "financial institution" and "financial" to "financial service provider". Combining mental health and mental health personnel.Changing "no relation" to "no relationship".
+
+### Question:
+  mandated reporter - ? (could be many, i.e. social worker, medical personnel, mh personnel)
+  paid caregiver - another where I am unsure, maybe professional service provider?
+  no APS referral, not applicable, no - (no might be column shift) but these, change to unknown? or is it interesting that some didn't get referred?
+```{r}
+demo_dat <- demo_dat %>% 
+  mutate(reporting_source_recode = case_when(str_detect(reporting_source, "data not|refused|doesn't know|#|not applicable") ~ "unknown",
+                                             str_detect(reporting_source, "sel|client") ~"self",
+                                             str_detect(reporting_source, "relative|family") ~ "family member",
+                                             str_detect(reporting_source, "community pro") ~"other community professional",
+                                             str_detect(reporting_source, "financial") ~ "financial service provider",
+                                             str_detect(reporting_source, "mental health") ~ "mental health personnel",
+                                             str_detect(reporting_source, "no relatio") ~"no relationship",
+                                             str_detect(reporting_source, "workshop|mandated report") ~ "flag",
+                                             is.na(reporting_source) ~ "unknown",
+                                             
+                                             TRUE ~ reporting_source))
+```
+
+Check for reporting source
+```{r}
+demo_dat %>% 
+  count(reporting_source, reporting_source_recode)
+```
+
+### Question: for previous APS involvement - seeing some responses that appear to fit in intervention, will this affect processing (given these are now subsets of data?) Hopefully this is just a column issue, but want to flag. 
+
```

Comment:
Good question--if it gets fixed in the next data dump then no need to worry but if it's a frequent miss entry we should expect, we will need to do some upstream clean up before we partition the data.